test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -163,36 +186,19 @@ public void lraCoordinatorShortTimeoutLRA(@ArquillianResource @OperateOnDeployme
 
         Assert.assertEquals("LRA participant action", 200, response.getStatus());
 
+        // The LRA transaction could have been started via lraClient but it is useful to
+        // test the filters as well
+        lraId = getFirstLRAFromFS();
+        assertNotNull("A new LRA should have been added to the object store before the JVM was halted.", lraId);
+        // wait almost until the LRA is timed out)
+        doWait((LRAListener.LRA_SHORT_TIMELIMIT * 1000) - 500);
         // Restarts lra-coordinator to simulate a crash
         restartContainer(LRA_COORDINATOR_CONTAINER_QUALIFIER);
 
-        // Waits for a period of time longer than the timeout of the LRA Transaction
-        doWait((LRAListener.LRA_SHORT_TIMELIMIT + 1L) * 1000);
-
-        // The LRA transaction could have been started via lraClient but it is useful to test the filters as well
-        lraId = getFirstLRAFromFS();
-        assertNotNull("A new LRA should have been added to the object store before the JVM was halted.", lraId);
         lraId = String.format("%s/%s", lraClient.getCoordinatorUrl(), lraId);
-
-        // Waits for a period of time longer than the timeout of the LRA Transaction
-        doWait((LRAListener.LRA_SHORT_TIMELIMIT + 1L) * 1000);
-
         // Checks recovery
         LRAStatus status = getStatus(new URI(lraId));
-
         LRALogger.logger.infof("%s: Status after restart is %s%n", status == null ? "GONE" : status.name());
-
-        if (status == null || status == LRAStatus.Cancelling) {
-            int sc = recover();
-
-            if (sc != 0) {
-                recover();
-            }
-        }
-
-        // LRA with short timeout should have timed out and cancelled
-        status = getStatus(new URI(lraId));
-
```

Comment:
I understand why this is being removed - it's the same reason as https://github.com/jbosstm/lra/pull/27/files#r1910235210

Reply:
But I am wondering if it might be useful to add the replacement you made there (or something similar)  because the `doWait((LRAListener.LRA_SHORT_TIMELIMIT * 1000) - 500);` (on line 194) is not long enough to timeout the LRA and even though when combined with the time `restartContainer(LRA_COORDINATOR_CONTAINER_QUALIFIER);` (from 196) takes, it is probably long enough to timeout the LRA, it might not be and so the `LRAStatus status = getStatus(new URI(lraId));` on (now) line 200 might find the LRA Cancelling still and so the assert on 202-203 might fail?